### DISCUSSION TITLE
more detail needed for the get started

### DISCUSSION THREAD

**Discussion by Pixouls** [BuchholzTim/Whitebird#84] [Setup Help]
### Describe your Issue
Hi, I'm a student and I want to move off of miro. I saw someone share this on mastodon. However, I have a really limited technology background, like I barely know how to use a terminal, unfortunately. I was able to clone the repo using github desktop (I don't really know how to use git otherwise) and see the files, but I'm stuck on the docker part to actually get the app to run. It's a .yml file, which I'd never encountered before. I wasn't sure what was going on so I tried installing docker, since online it says it should include docker compose, and running the command in the get started in my terminal but wasn't really getting anywhere. I'm just really confused, especially since I know nothing about docker.

### How to reproduce?
N/A

### What would be the expected behavior?
If this were something I wanted to get other students or my professors on, it would be helpful to have more detail on the get started on how you would imagine someone who has very little background in technology trying to set this up from scratch. I know it's work and I'm very sorry but I'd really like to try to use your tool as well.

### Tell us about your client:
- Browser: Ungoogled Chromium
- OS: Windows 10

**Comment by Pixouls** [comment]
I also found my docker install was not running properly because i did not have an additional WSL install, but now I am confused if I should have done the clone with docker as well? i still can't figure out how I'm suppose to run the compose command yet either.

**Comment by Pixouls** [comment]
Ok so here's what I ended up doing.
I didn't know how to navigate to the right directory. I followed https://www.howtogeek.com/659411/how-to-change-directories-in-command-prompt-on-windows-10/#:~:text=If%20the%20folder%20you%20want,window%2C%20and%20then%20press%20Enter.&text=The%20directory%20you%20switched%20to%20will%20be%20reflected%20in%20the%20command%20line.
To get to where I had cloned the files using github desktop
`cd "C:\Users\Pixel\Documents\GitHub\Whitebird\docker\whitebird_prod'`
and then i ran the command provided
`docker-compose up -d`

Finally, after a few minutes, it starts unpacking whatever and I can get to localhost. 

Hopefully, my explaining my confusion will help someone else.

**Reply by MarcBollmann** [reply]
you did everything right if there any further questions let us know!
